### PR TITLE
debug mode: silence warning, print config

### PR DESCRIPTION
**Pull request by jgehrcke**
Two tiny improvements, see commit messages. Thanks for review!

### CODE REVIEW

File: conbench/__init__.py [hunk 1]
```diff
@@ -14,7 +15,15 @@ def create_application(config):
 
     application = f.Flask(__name__)
     application.config.from_object(config)
+
     _init_application(application)
+
+    if application.config["DEBUG"]:
+        c = json.dumps(application.config, sort_keys=True, default=str, indent=2)
```
Comment (jonkeane) [inline conbench/__init__.py]: I don't think I knew that `json.dumps` had such convenience bits as sorting and indents ❤️

File: .flaskenv [hunk 2]
```diff
@@ -4,6 +4,7 @@ DB_PASSWORD=postgres
 DB_PORT=5432
 FLASK_APP=conbench
 FLASK_ENV=development
+FLASK_DEBUG=true
```
Comment (jonkeane) [inline .flaskenv:7]: I'm 95% certain this `.flaskenv` is not used in production deployments, so it's all good to hard code debug to true here — but want to double check that you agree that's actually the case here

**Review by jonkeane — APPROVED** [review]
Thank you for this clean up! One small question

File: .flaskenv (hunk 2 above)
Comment (jgehrcke) [reply .flaskenv:7]: Oh. Yeah great question. If `.flaskenv` is used in production then debug mode was already enabled so far via the `FLASK_ENV=development` setting (it's deprecated, but still honored). That's why I thought that the change won't be a regression. We should certainly double-check, but I like that you are already 95 % confident :).

File: conbench/__init__.py (hunk 1 above)
Comment (jgehrcke) [reply conbench/__init__.py]: yeah! :) True magic here is the `default=str` which is a meaningful fallback for objects that are not JSON-serializable, such as `datetime.timedelta` objects (needed that here).

**Comment by jgehrcke** [comment]
CI found that I committed before cleaning up:
```
./conbench/__init__.py:22:9: F841 local variable 'c' is assigned to but never used
```

update: added a new commit and squashed, force-pushed right away -- let's re-run CI

**Review by austin3dickey — APPROVED** [review]

**Comment by jgehrcke** [comment]
I'm going ahead and merge this, let's iterate if needed. Thanks for the reviews!

**Pull request merged**